__tests__/header.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import Header from '../src/core/adapt-contrib-core/templates/header';
+import { act } from 'react-dom/test-utils';
+import { afterEach, beforeEach, describe, expect, it, jest } from '@jest/globals';
+import device from 'core/js/device';
+import { unmountComponentAtNode, render } from 'react-dom';
+
+let container = null;
+beforeEach(() => {
+  jest.clearAllMocks();
+  // setup a DOM element as a render target
+  container = document.createElement('div');
+  document.body.appendChild(container);
+});
+
+afterEach(() => {
+  // cleanup on exiting
+  unmountComponentAtNode(container);
+  container.remove();
+  container = null;
+});
+
+describe('Header Component State', () => {
+  it('should display the header body', () => {
+    const body = 'test body text';
+    const props = {
+      body
+    };
+    act(() => {
+      render(<Header {...props} />, container);
+    });
+    expect(container.textContent).toBe(body);
+  });
+
+  it('should display the header instructions', () => {
+    const instruction = 'test instructions';
+    const props = {
+      instruction
+    };
+    act(() => {
+      render(<Header {...props} />, container);
+    });
+    expect(container.textContent).toBe(instruction);
+  });
+
+  it('should be able to set the displayTitle', () => {
+    const displayTitle = 'test body text';
+    const props = {
+      displayTitle
+    };
+    act(() => {
+      render(<Header {...props} />, container);
+    });
+    expect(container.textContent).toBe('');
```

Comment:
Should this be:
```suggestion
    expect(container.textContent).toBe(displayTitle);
```

Reply:
The displayTitle is not compiled in the same way the other props are using `dangerouslySetInnerHTML` so Im not sure there is a way to assert the displayTitle is shown.

Reply:
ah right, thanks. In that case it may be best to remove this and perhaps add a TODO
We don't want to mislead that we have coverage to check displayTitle when we aren't really doing that